### ISSUE TITLE
Cannot sign in with account 

### ISSUE THREAD

**Issue by csangani** [yuliskov/SmartTube#1328]
I'm unable to sign into my account on both the stable and beta versions. On my TV, the code shows up fine. When I enter the code on YouTube.com/activate on my phone, I get the following error:

400. That’s an error.

Error: invalid_request

state_wrapper is needed.

**Comment by codex704** [comment]
Yeah I reported it to the dev on the 19th but unfortunately the Dev is from/in Ukraine so the project is on hold until further notice

**Comment by vaderag** [comment]
Same issue here. Hope the dev is okay

**Comment by hotn** [comment]
I had the same issue when using my phone's browser, but it worked when using my desktop browser.

**Comment by vaderag** [comment]
Yes, I believe this is due to something to do with needing to use Chrome. I was using Edge on my phone but when I tried chrome it worked!

**Comment by GnJCh** [comment]
The solution is to log in from Google Chrome either on Android PC or iOS but it has to be Google Chrome.

**Comment by HSchultz83** [comment]
I'm just here to say I had the exact issue an confirm chrome works firs time for some reason. I'm on an android phone

**Comment by Midguett** [comment]
> I had the same issue when using my phone's browser, but it worked when using my desktop browser. 

Worked like a charm! Thanks!

**Comment by AutomaticFailure** [comment]
I've tried various browsers and uninstalling/reinstalling the app. Nothing seems to work to allow me to log in anymore. Using latest beta 18.28
